### PR TITLE
Add create-virtual-file-system-in-dotnet.yml

### PR DESCRIPTION
**Pull request by jtothej**
Add create-virtual-file-system-in-dotnet.yml

### CODE REVIEW

File: host-interaction/file-system/create-virtual-file-system-in-dotnet.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+rule:
+  meta:
+    name: create virtual file system in .NET
```
Comment (williballenthin) [inline host-interaction/file-system/create-virtual-file-system-in-dotnet.yml:3]: this must collect some very interesting samples 🤔

File: host-interaction/file-system/create-virtual-file-system-in-dotnet.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    scope: function
+    att&ck:
+      - Defense Evasion::Hide Artifacts::Hidden File System [T1564.005]
+    references:
+      - https://learn.microsoft.com/en-us/dotnet/api/system.web.hosting.virtualpathprovider?view=netframework-4.8.1
+    examples:
+      - be619015f1d5fb13d3bcd48d1b0843759a5e040d8af3da50f2d8f55b5a680e02:0x600000F
+  features:
+    - or:
+      - api: System.Web.Hosting.VirtualPathProvider::ctor
```
Comment (williballenthin) [inline host-interaction/file-system/create-virtual-file-system-in-dotnet.yml:16]: consider adding `format: dotnet`

Comment (jtothej) [reply host-interaction/file-system/create-virtual-file-system-in-dotnet.yml:17]: Thanks! Fixed in https://github.com/mandiant/capa-rules/pull/783/commits/2e0b272eac96f630cd6caaa8aaab3bddbba83847

**Comment by mr-tz** [comment]
thank you!

**Pull request merged**